shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java

```diff
@@ -122,6 +133,7 @@ public void testNotifyBridgeStatusChange() throws InterruptedException {
     }
 
     @Test
+    @WithPrometheus
```

Comment:
Why this change?

Reply:
Because I found out that this is currently working just because of luck :) If you execute the test standalone it does not work because Prometheus is not installed on the mock.
This is currently working because the test `testBridgesAreDeployed` is annotated with `WithPrometheus` and is executed *before* this test: all the tests that are executed after  `testBridgesAreDeployed` have prometheus installed as well.